README.md

```diff
@@ -37,9 +37,16 @@ And all of this will be built in a beautiful and well-organized way!
 
 ### Key Features <a name="key-features"></a>
 
-- **Add new students or teachers**
+- **Add new students**
+- **Add new teachers**
 - **Add new books**
+- **Add new rental**
 - **Save records of who borrowed a given book and when**
+- **Store data in JSON**
+
+### Upcoming Features <a name="Upcoming-features"></a>
+
+- **Unit Tests**
 
```

Comment:
- [ ] @Team Great work on your license file, however, the license link is broken, this is because you have referenced the license file wrongly instead of `./LICENSE/MIT.md` it should be `./LICENSE`